### ISSUE TITLE
Include a buttons for row and column headers in the toolbar that appears when editing a table.

### ISSUE THREAD

**Issue by nikkimk** [tinymce/tinymce#5206]
**Is your feature request related to a problem? Please describe.**
Accessibility: Tables should include semantic markup for headers, and making a table that is accessible should be easy as it can be. (See https://webaim.org/techniques/tables/data#th)

**Describe the solution you'd like**
There should be a toggle button that turns the first row into a <thead/> row headers as well as a toggle that turns the first column into headers.

**Describe alternatives you've considered**
Any other alternative would make it easy for the user to miss that this is an option.

**Comment by lnewson** [comment]
Hi @nikkimk,

This feature was added in TinyMCE 5.9 and can be used by adding the `tablerowheader` and `tablecolheader` toolbar buttons. More information can be found here: https://www.tiny.cloud/docs/release-notes/release-notes59/#newtabletoolbarbuttonsandmenuitems
